src/gmail.js

```diff
@@ -3932,6 +3932,33 @@ var Gmail = function(localJQuery) {
             return el.html();
         },
 
+        /**
+            Get the email attachments
+        */
+        attachments: function() {
+            var out = [];
+            var failed = false;
+
+            this.dom("attachments").each(function() {
+                var el = $(this);
+
+                var result = {};
+                result.$el = el;
+                result.name = el.find("div.vI").html();
+                result.size = el.find("div.vJ").html();
```

Comment:
From the outside as an API consumer, this property is easy to assume to be numerical, but looking at the code, it looks like its a displayvalue ("5mb", etc).

Could we name it better to avoid incorrect usage?

Reply:
I think this is ok.  The return type is GmailDomAttachment where size is a string.  This matches what gmail.dom.email.attachments() returns

Reply:
@ek23 @josteink I am having google chrome extension , in which I want to get the attachment of the email and want to download those into my system. is this possible to get URL of the attachment via your API

Reply:
Yes. Check api.new.email_data(). It has information about extensions, including URL.

Reply:
@josteink can you please give me a example code as I tried to ejected your gmail.js into my DOM but it returns nothing , little guidance will be appreciated

Reply:
Use the supplied boilerplate project. 

Load Gmail and a specific email thread. Go to the web CI sile and type:

    var attachements = gmail.new.get.email_data().attachments;